tests/integration_app_test.py

```diff
@@ -0,0 +1,58 @@
+import os
+import pytest
+from dotenv import load_dotenv, find_dotenv
+import requests
+from todo_app import app
+
+
+@pytest.fixture
+def client():
+    # Use test environment config here
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+
+    # Create new app instance
+    test_app = app.create_app()
+
+    # Create client using the new app instance
+    with test_app.test_client() as client:
+        yield client
+
+
+class StubResponse():
+    def __init__(self, fake_response_data):
+        self.fake_response_data = fake_response_data
+        self.status_code = 200
+
+    def json(self):
+        return self.fake_response_data
+
+
+def get_mock_trello(url, params={}):
+    # The expected URL should match the URL format used in your application
+    expected_url = f'https://api.trello.com/1/boards/64f6582da14testb49e12999999/lists'
```

Comment:
Minor, but pulling in the board id as a variable might both:
* Make this more readable
* Be a bit more flexible when someone inevitably decides to change it!
```suggestion
    test_board_id = os.getenv("BOARD_ID")
    expected_url = f'https://api.trello.com/1/boards/{test_board_id}/lists'
```